Source/Core/Core/Analytics.cpp

```diff
@@ -147,12 +243,46 @@ constexpr std::array<const char*, 14> GAME_QUIRKS_NAMES{"icache-matters",
                                                         "mismatched-gpu-colors-between-xf-and-bp",
                                                         "uses-uncommon-wd-mode",
                                                         "uses-wd-unimplemented-ioctl"};
+constexpr bool ValidateQuirkMatchesName(GameQuirk quirk, std::string_view name)
```

Comment:
```suggestion
static constexpr bool ValidateQuirkMatchesName(GameQuirk quirk, std::string_view name)
```

otherwise this has external linkage

Reply:
Oops, good catch!

While I'm at it is there any reason I shouldn't add static to GAME_QUIRK_NAMES?

Reply:
const or constexpr on variables at file scope makes them internally linked by default, so it's not necessary